src/components/Report/Report.tsx

```diff
@@ -65,6 +67,25 @@ interface ReportProperties {
     onClose?: () => void;
 }
 
+// see node PUBLIC_GAMEDATA_FIELDS in the backend
+type Gamedata = {
+    outcome: string;
+    winner: number;
+};
+
+function checkEscapingForResignation(
+    game_id?: number,
+    reported_user_id?: number,
+): Promise<string | null> {
+    return get(`/termination-api/game/${game_id}`).then((gamedata: Gamedata) => {
+        if (gamedata?.outcome.includes("Resignation") && gamedata.winner !== reported_user_id) {
+            return "That player resigned, so 'stopped playing' doesn't make sense. Do you mean 'sandbagging'?";
```

Comment:
Missing a `_(..)`

But also, would it be better to just leave it as a statement and not a sort of hint that they should just file their grief as a sandbagging report?

Reply:
Doh, oops!   😜

I put the bit about sandbagging, because that has often seemed to be what it is, but you're probably right, I'll get rid of that with the `_()`

Reply:
Either way about the sandbagging, I trust your intuition more than my own on these things these days.